modules/@apostrophecms/color-field/ui/apos/components/AposColor.vue

```diff
@@ -97,7 +97,7 @@
           class="apos-color__presets-color"
           @click="handlePreset(c)"
         >
-          <Checkboard />
+          <AposColorCheckerboard />
```

Comment:
This component now lies in `modules/@apostrophecms/ui/ui/apos/components/AposColorCheckerboard.vue`, we don't need to explicitly import it

Reply:
Is it used in another location? Not sure it makes sense to make it available from everywhere